src/bakery.js

```diff
@@ -1,7 +1,13 @@
 export default class Bakery {
 
-  static bake() {
-    document.cookie = 'smartbanner_exited=1';
+  static bake(daysToHide) {
```

Comment:
I understand what you're trying to do here, but let's keep it to milliseconds without any handling. Robust, less error-prone and flexible. Most frameworks that will work with `smartbanner.js` (we've seen Rails etc.) will be able to parse in `ms` unit nicely and can then handle their time on the CMS or configuration side.